tools/pythonpkg/tests/fast/pandas/test_raise_warning.py

```diff
@@ -0,0 +1,61 @@
+import duckdb
+import pandas as pd
```

Comment:
This needs `pd = pytest.importorskip("pandas")

Reply:
Oh we removed pandas from the test dependencies? Was not aware of that, will change it to importorskip